checkov/terraform/checks/provider/aws/defaulttags.py

```diff
@@ -0,0 +1,29 @@
+from typing import Dict, List, Any
+
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.provider.base_check import BaseProviderCheck
+
+
+class AWSDefaultTags(BaseProviderCheck):
+    def __init__(self) -> None:
+        name = "Ensure AWS provider uses default_tags"
+        id = "CKV_AWS_166"
```

Comment:
@bdausses Please use an unused check ID, this one is already taken and it fails the tests